### PR TITLE
[WICKET-6617] headers are added to header-items if specified

### PR DESCRIPTION
**Pull request by solomax**

### CODE REVIEW

File: wicket-core/src/main/java/org/apache/wicket/markup/html/internal/HtmlHeaderContainer.java [hunk 1]
```diff
@@ -161,19 +161,19 @@ public final void onComponentTagBody(MarkupStream markupStream, ComponentTag ope
...
-			// Render the header sections of all components on the page
-			AbstractHeaderRenderStrategy.get().renderHeader(this,
-				new HeaderStreamState(markupStream, openTag), getPage());
+				// Render the header sections of all components on the page
+				AbstractHeaderRenderStrategy.get().renderHeader(this,
+					new HeaderStreamState(markupStream, openTag), getPage());
 
-			// Close the header response before rendering the header container itself
-			// See https://issues.apache.org/jira/browse/WICKET-3728
-			headerResponse.close();
```
[the remark this answers is not in the document]
Comment (solomax) [reply wicket-core/src/main/java/org/apache/wicket/markup/html/internal/HtmlHeaderContainer.java]: `close()` is being called automatically by `try with resource` statement
I'll modify comment

[the remark this answers is not in the document]
Comment (svenmeier) [reply wicket-core/src/main/java/org/apache/wicket/markup/html/internal/HtmlHeaderContainer.java]: Thanks understood :)

File: wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js [hunk 2]
```diff
@@ -2116,10 +2116,15 @@
 
 			// Adds the element to page head
 			addElement: function (element) {
-				var head = document.getElementsByTagName("head");
+				var headItems = document.querySelector('head meta[name="wicket.header.items"]');
```
Comment (svenmeier) [inline wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: Seems we haven't used querySelector() before - are we increasing the minimum supported browser version with this?

Comment (solomax) [reply wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: According to this table [1] `querySelector` is being supported by all browsers ...

[1] https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector#Browser_Compatibility

**Comment by solomax** [comment]
@svenmeier  Should this be merged into 7.x?

**Comment by solomax** [comment]
OK :)

**Pull request closed without merging**